contrib/clojure-package/examples/gan/src/gan/viz.clj

```diff
@@ -18,14 +18,9 @@
 (ns gan.viz
```

Comment:
I ran the GAN example and everything looks great 💯  - I even think it's faster 🚗 
Nice job.